### PR TITLE
Log the indexes when using st_b0shim max-intensity

### PR DESCRIPTION
**Pull request by po09i**
## Description
Display to the terminal the resulting indexes

## Linked issues
https://github.com/shimming-toolbox/fsleyes-plugin-shimming-toolbox/pull/51

### CODE REVIEW

File: shimmingtoolbox/cli/b0shim.py [hunk 1]
```diff
@@ -1107,15 +1107,19 @@ def max_intensity(fname_input, fname_mask, fname_output, verbose):
         nii_mask = nib.load(fname_mask)
 
     # Shim
-    index_per_slice = shim_max_intensity(nii_input, nii_mask)
+    # Output with 1 index
+    index_per_slice = shim_max_intensity(nii_input, nii_mask) + 1
+
+    # Log the output (1 index)
+    logging.info(f"Max intensity indexes: {index_per_slice}")
```
Comment (4rnaudB) [inline shimmingtoolbox/cli/b0shim.py]:
```suggestion
    logger.info(f"Max intensity indexes: {index_per_slice}")
```

Comment (4rnaudB) [reply shimmingtoolbox/cli/b0shim.py]: The index do not appear but the directory does (line 1120).

**Review by 4rnaudB — COMMENTED** [review]
Also the base directory chosen for the output is in shimming-toolbox and is not in .gitignore, is it wanted?

File: shimmingtoolbox/cli/b0shim.py (hunk 1 above)
Comment (po09i) [reply shimmingtoolbox/cli/b0shim.py]: Good catch

**Comment by po09i** [comment]
The default path is the current directory

File: shimmingtoolbox/cli/b0shim.py (hunk 1 above)
Comment (4rnaudB) [reply shimmingtoolbox/cli/b0shim.py]: Still nothing, i tried modifying  line 1120 output but nothing. Does this function get run from fsleyes?

Comment (po09i) [reply shimmingtoolbox/cli/b0shim.py]: Not yet, this is just in ST, I'm waiting for this to be merged to update fsleyes

Comment (4rnaudB) [reply shimmingtoolbox/cli/b0shim.py]: image

Comment (po09i) [reply shimmingtoolbox/cli/b0shim.py]: Screen Shot 2022-11-29 at 5 05 11 PM

Comment (4rnaudB) [reply shimmingtoolbox/cli/b0shim.py]: Peut-être tester avec une deuxième personne qui utilise Linux?

**Review by 4rnaudB — APPROVED** [review]
Works when using make install

File: shimmingtoolbox/cli/b0shim.py (hunk 1 above)
Comment (po09i) [reply shimmingtoolbox/cli/b0shim.py]: We needed to make install

**Pull request merged**